twitter_scrooge/twitter_scrooge.bzl

```diff
@@ -9,7 +9,7 @@ load("//scala:scala.bzl",
 def twitter_scrooge():
   native.maven_server(
     name = "twitter_scrooge_maven_server",
-    url = "http://mirror.bazel.build/repo1.maven.org/maven2/",
+    url = "http://repo1.maven.org/maven2/",
```

Comment:
Why can't we use the Bazel mirror?

Reply:
i can't remember the reason to switch, switched back